res/database/Default/user/cujo.xml

```diff
@@ -0,0 +1,1830 @@
+		<news id="cujo-news-brettspiel-1985" type="0" thread_id="cujo-news-brettspiel-1985" creator="8827" created_by="Cujo">
+			<title>
+				<de>Detektivspiel ist das Brettspiel des Jahres 1985</de>
+				<en>Detective game is voted board game of the year 1985</en>
+			</title>
+			<description>
+				<de>"Dr. Watson Kriminalzimmer" wurde zum Brettspiel des Jahres 1985 gekürt. Der Jury imponierte, dass das Spiel sowohl allein als auch in einer Gruppe gespielt werden kann.</de>
+				<en>"Dr. Watson Crime Room" was chosen as the board game of the year 1985. The jury was impressed by the fact that the game can be played both alone and in a group.</en>
+			</description>
+			<data genre="3" price="0.49" quality="45" />
+			<availability year_range_from="1985" year_range_to="1985" />
+		</news>
+		<news id="cujo-news-brettspiel-1990" type="0" thread_id="cujo-news-brettspiel-1990" creator="8827" created_by="Cujo">
+			<title>
+				<de>Jury kürt Brettspiel des Jahres 1990</de>
+				<en>Jury selects board game of the year 1990</en>
+			</title>
+			<description>
+				<de>Das Brettspiel des Jahres 1990 heißt "Blaues Blut wird zur Pflicht". Mehrere Barone kämpfen in Auktionen um die besten Antiquitäten. Um ihr Ziel zu erreichen, können sie Diebe und Spione verpflichten.</de>
+				<en>The board game of the year 1990 is called "Blue Blood Becomes Duty". Several barons fight in auctions for the best antiques. To achieve their goal, they can hire thieves and spies.</en>
+			</description>
+			<data genre="3" price="0.48" quality="46" />
+			<availability year_range_from="1990" year_range_to="1990" />
+		</news>
+		<news id="cujo-news-brettspiel-1995" type="0" thread_id="cujo-news-brettspiel-1995" creator="8827" created_by="Cujo">
+			<title>
+				<de>Das Brettspiel des Jahres 1995 ist ein Aufbauspiel</de>
+				<en>The board game of the year 1995 is a building game</en>
+			</title>
+			<description>
+				<de>"Die Pioniere von Kastlan" wurde zum Brettspiel des Jahres 1995 gewählt. Die Spieler müssen Rohstoffe sammeln, um damit Straßen, Siedlungen und Städte bauen zu können. Erfunden wurde es von dem Spieleentwickler Karl Stäuber.</de>
+				<en>"The Pioneers of Kastlan" was voted board game of the year in 1995. Players have to collect raw materials in order to build roads, settlements and cities. It was invented by the game developer Karl Stäuber.</en>
+			</description>
+			<data genre="3" price="0.47" quality="47" />
+			<availability year_range_from="1995" year_range_to="1995" />
+		</news>
+		<!-- "Gamezworld AG" -->
+		<news id="cujo-news-gamezworld-01" type="0" thread_id="cujo-news-tvt" creator="8827" created_by="Cujo">
+			<title>
+				<de>TV Tower-Entwickler arbeiten im Home-Office</de>
+				<en>TV Tower developers work in a home office</en>
+			</title>
+			<description>
+				<de>Wegen der Pandemie arbeiten ab sofort alle TV Tower-Entwickler im Home-Office. Laut der Gamezworld AG sind weltweit etwa 35.000 Mitarbeiter davon betroffen.</de>

```

Comment:
Variablen: Mitarbeiteranzahl, weltweit/nur in Deutschland... vielleicht auch nicht nur Entwickler sondern auch die Spieler

zumindest wäre Abwechslung denkbar